### PR TITLE
Add convert rules related to `torch.nn.Lazy*`

### PR DESCRIPTION
**Pull request by guozixu2001**
Add convert rules related to `torch.nn.Lazy*`
```python
torch.nn.LazyConv1d
torch.nn.LazyConv2d
torch.nn.LazyConv3d
torch.nn.LazyConvTranspose1d
torch.nn.LazyConvTranspose2d
torch.nn.LazyConvTranspose3d
torch.nn.LazyLinear
torch.nn.LazyBatchNorm1d
torch.nn.LazyBatchNorm2d
torch.nn.LazyBatchNorm3d
torch.nn.LazyInstanceNorm2d
torch.nn.LazyInstanceNorm1d
torch.nn.LazyInstanceNorm3d
```

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LazyBatchNorm1d.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+| ------------- | ------------ | ------------------------------------------------------ |
+| -             | num_features   | 表示输入 Tensor 通道数，PyTorch 无此参数，Paddle 需要根据实际输入 Tensor 的通道数进行设置。                             |
+| eps           | epsilon      | 为了数值稳定加在分母上的值，仅参数名不一致。                                                                                                      |
+| momentum      | momentum      | 表示归一化函数中的超参数, PyTorch 和 Paddle 公式实现细节不一致，两者正好是相反的，需要转写。                                                                     |
+| -             | weight_attr  | 指定权重参数属性的对象。如果为 False, 则表示每个通道的伸缩固定为 1，不可改变。默认值为 None，表示使用默认的权重参数属性。                                                        |
+| -             | bias_attr    | 指定偏置参数属性的对象。如果为 False, 则表示每一个通道的偏移固定为 0，不可改变。默认值为 None，表示使用默认的偏置参数属性。                                                       |
+| -             | data_format  | 指定输入数据格式，PyTorch 无此参数，Paddle 保持默认即可。                                                                                        |
+| affine        | -                | 是否进行反射变换， Paddle 无此参数，需要转写。                                                                                                 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LazyBatchNorm1d.md:32]: 这一类应该都可以都换一个写法，把affine对应到weight_attr、bias_attr

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LazyBatchNorm1d.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+| dtype         | -            | 指定权重参数属性的对象，一般对网络训练结果影响不大，可直接删除。 |
+
+### 转写示例
+
+#### num_features: 输入通道数
+在 PyTorch 中，使用 `LazyBatchNorm1d` 时可以不指定 `num_features`，它会在第一次前向传播时根据输入 Tensor 的形状自动确定；而在 Paddle 中，创建 `BatchNorm1D` 时必须明确指定 `num_features` 参数，其值应与输入 Tensor 的通道数保持一致。
+```python
+# PyTorch 写法
+bn = torch.nn.LazyBatchNorm1d()
+input = torch.randn(3, 5)  # 5 是输入通道数
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LazyBatchNorm1d.md]: BatchNorm1d是不是得输入3D？NCL

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LazyConv1d.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| -             | in_channels            | 表示输入 Tensor 通道数，PyTorch 无此参数，Paddle 需要根据实际输入 Tensor 的通道数进行设置。                           |
+| out_channels  | out_channels            | 表示输出 Tensor 通道数。                           |
+| kernel_size   | kernel_size            | 表示卷积核大小。                           |
+| stride        | stride            | 表示卷积核步长。                           |
+| padding          | padding            | 表示填充大小。                           |
+| dilation          | dilation            | 表示空洞大小。                           |
+| groups          | groups            | 表示分组数。                           |
+| bias         | -            | 是否在输出中添加可学习的 bias， Paddle 无此参数，需要转写。                           |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LazyConv1d.md:36]: 这一类应该也能改个写法，bias可以对应到bias_attr

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LazyInstanceNorm1d.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+### 参数映射
+
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| -             | num_features   | 表示输入 Tensor 通道数，PyTorch 无此参数，Paddle 需要根据实际输入 Tensor 的通道数进行设置。  |
+| eps           | epsilon       | 为了数值稳定加在分母上的值。             |
+| momentum      | momentum      | 此值用于计算 moving_mean 和 moving_var，值的大小 Paddle = 1 - PyTorch，需要转写。        |
+| affine        | -             | 是否使用可学习的仿射参数，Paddle 无此参数。可通过 weight_attr 和 bias_attr 控制。              |
+| track_running_stats | -       | 是否跟踪运行时的 mean 和 var， Paddle 无此参数。暂无转写方式。  |
+| dtype         |  -            | 输出数据类型， Paddle 无此参数，一般对网络训练结果影响不大，可直接删除。 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LazyInstanceNorm1d.md]: 如果按可以直接删除，后面就不写转写方式了吧

**Review by zhwesky2010 — COMMENTED** [review]
整体排查
1. affine对应到weight_attr、bias_attr的写法
2. bias对应到bias_attr的写法
3. dtype如果按 **可删除**，就无需转写方式
4. 转写示例的forward输入维度是否符合要求

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LazyBatchNorm1d.md (hunk 1 above)
Comment (guozixu2001) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LazyBatchNorm1d.md:32]: 这里全部是参考的已有文档的写法，现有很大一批文档都是这么写的，比如：docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Conv1d.md。
沿用之前的这个写法是担心不一致会让用户产生疑惑。

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LazyInstanceNorm1d.md (hunk 4 above)
Comment (guozixu2001) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LazyInstanceNorm1d.md:23]: 下面的都是参考 doc/docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.BatchNorm1d.md 写的，那原先这个也修改了？

**Review by zhwesky2010 — APPROVED** [review]
LGTM
